diff --git a/src/CommerceAutoSkuManager.php b/src/CommerceAutoSkuManager.php
index 36c8a93..3b27e13 100644
--- a/src/CommerceAutoSkuManager.php
+++ b/src/CommerceAutoSkuManager.php
@@ -112,7 +112,7 @@ class CommerceAutoSkuManager implements CommerceAutoSkuManagerInterface {
    * @return bool
    *   True if the entity has a label property.
    */
-  public function hasSku() {
+  public function hasSku(): bool {
     /** @var \Drupal\Core\Entity\EntityTypeInterface $definition */
     $definition = $this->entityTypeManager->getDefinition($this->entity->getEntityTypeId());
     // @todo cleanup.
@@ -125,6 +125,7 @@ class CommerceAutoSkuManager implements CommerceAutoSkuManagerInterface {
     if (isset($fields['sku'])) {
       return TRUE;
     }
+    return FALSE;
   }
 
   /**
diff --git a/src/CommerceAutoSkuPermissionController.php b/src/CommerceAutoSkuPermissionController.php
index 7528032..82e0a16 100644
--- a/src/CommerceAutoSkuPermissionController.php
+++ b/src/CommerceAutoSkuPermissionController.php
@@ -41,7 +41,7 @@ class CommerceAutoSkuPermissionController implements ContainerInjectionInterface
    * {@inheritdoc}
    */
   public static function create(ContainerInterface $container) {
-    return new static($container->get('entity_type.manager'));
+    return new self($container->get('entity_type.manager'));
   }
 
   /**
diff --git a/src/Form/CommerceAutoSkuForm.php b/src/Form/CommerceAutoSkuForm.php
index 0d55e24..282feec 100644
--- a/src/Form/CommerceAutoSkuForm.php
+++ b/src/Form/CommerceAutoSkuForm.php
@@ -66,7 +66,7 @@ class CommerceAutoSkuForm extends FormBase {
    * {@inheritdoc}
    */
   public static function create(ContainerInterface $container) {
-    return new static (
+    return new self(
       $container,
       $container->get('plugin.manager.commerce_autosku_generator')
     );
diff --git a/src/Plugin/CommerceAutoSkuGenerator/Token.php b/src/Plugin/CommerceAutoSkuGenerator/Token.php
index ebebe5f..afaa792 100644
--- a/src/Plugin/CommerceAutoSkuGenerator/Token.php
+++ b/src/Plugin/CommerceAutoSkuGenerator/Token.php
@@ -52,7 +52,7 @@ class Token extends CommerceAutoSkuGeneratorBase {
    * {@inheritdoc}
    */
   public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
-    return new static(
+    return new self(
       $configuration,
       $plugin_id,
       $plugin_definition,
diff --git a/src/Plugin/Derivative/CommerceAutoSkuConfigTask.php b/src/Plugin/Derivative/CommerceAutoSkuConfigTask.php
index c9a0b20..b73b52b 100644
--- a/src/Plugin/Derivative/CommerceAutoSkuConfigTask.php
+++ b/src/Plugin/Derivative/CommerceAutoSkuConfigTask.php
@@ -30,7 +30,7 @@ class CommerceAutoSkuConfigTask extends DeriverBase implements ContainerDeriverI
    * {@inheritdoc}
    */
   public static function create(ContainerInterface $container, $base_plugin_id) {
-    return new static(
+    return new self(
       $container->get('entity_type.manager')
     );
   }
