diff --git a/src/Plugin/Field/FieldFormatter/ImageZoomFormatter.php b/src/Plugin/Field/FieldFormatter/ImageZoomFormatter.php
--- a/src/Plugin/Field/FieldFormatter/ImageZoomFormatter.php
+++ b/src/Plugin/Field/FieldFormatter/ImageZoomFormatter.php
@@ -61,7 +61,7 @@
    * {@inheritdoc}
    */
   public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
-    return new static(
+    return new self(
       $plugin_id,
       $plugin_definition,
       $configuration['field_definition'],
