diff --git a/src/Form/ToolbarVisibilitySettingsForm.php b/src/Form/ToolbarVisibilitySettingsForm.php
--- a/src/Form/ToolbarVisibilitySettingsForm.php
+++ b/src/Form/ToolbarVisibilitySettingsForm.php
@@ -70,7 +70,7 @@
    * {@inheritdoc}
    */
   public static function create(ContainerInterface $container) {
-    return new static(
+    return new self(
       $container->get('config.factory'),
       $container->get('config.typed'),
       $container->get('theme_handler'),
